Sources/MapboxCoreNavigation/Resources/en.lproj/Localizable.strings

```diff
@@ -1,27 +1,3 @@
-/* Format for speech string after completing a maneuver and starting a new step; 1 = distance */
```

Comment:
Are these lines currently unused in the SDK?🤔

Reply:
Yes, it’s currently unused. The Directions API (online) and MapboxNavigationNative (offline) are responsible for providing speech strings.